### DISCUSSION TITLE
Процесс обхода сам останавливается

### DISCUSSION THREAD

**Discussion by korolyuga** [Anonym-tsk/nfqws-keenetic#135] [Q&A] [answered]
Здравствуйте! На странице [ip роутера]:90 нажимаю start выходит сообщение "NFQWS started" открываю в соседней вкладке например нельзяграм, страница открывается, но а далее ничего не грузится. Возвращаюсь на веб интерфейс nfqws и вижу что процесс остановился, т.е. проработал несколько секунд. Как это исправить?

p.s. keenetic giga

**Comment by Anonym-tsk** [comment]
**Выполните команды и приложите их вывод**
`opkg info nfqws-keenetic`

`/opt/etc/init.d/S51nfqws restart`

`cat /opt/etc/nfqws/nfqws.conf`

`ps | grep nfqws`

`iptables-save | grep 200`

**Reply by korolyuga** [reply]
~ # opkg info nfqws-keenetic
Package: nfqws-keenetic
Version: 2.6.5
Depends: iptables, busybox
Conflicts: tpws-keenetic
Status: install user installed
Section: net
Architecture: all
Size: 682369
Filename: nfqws-keenetic_2.6.5_all_entware.ipk
Conffiles:
 /opt/etc/nfqws/nfqws.conf d897b5561e5f36b8dc0304f407eef540157f76fc2789dfe61ed17af3c2d486ad
 /opt/etc/nfqws/user.list 45dc2adaa172b86d73369c6ed12a8a0e648b851b66293b11514c3b1d4bd3fce6
 /opt/etc/nfqws/auto.list e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
 /opt/etc/nfqws/exclude.list 3ffc3588b7ed7f4585a6563b186d6e57c0d19ba1276f6b8ec595ce015b1da562
Description: NFQWS service
Installed-Time: 1734079892
--------------------------------

 # /opt/etc/init.d/S51nfqws restart
Service NFQWS is not running
Policy 'nfqws' not found. All traffic will be processed.
github version v69.6 (7b057491afc2d8857dcdba00ea0c09991d09d665)

we have 3 user defined desync profile(s) and default low priority profile 0
Loading hostlist /opt/etc/nfqws/exclude.list
loading plain text list
Loaded 35 hosts from /opt/etc/nfqws/exclude.list
Started NFQWS service
----------------------------------------
 # ps | grep nfqws
 1220 nobody   13520 S    /opt/zapret/nfq/nfqws --user=nobody --dpi-desync-fwm
 2956 root      5400 S    grep nfqws
-----------------------------------------

# iptables-save | grep 200
-A POSTROUTING -o eth3 -p udp -m multiport --dports 443,50000:50099 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A POSTROUTING -o eth3 -p tcp -m multiport --dports 443 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A _NDM_IP_PROTECT -p tcp -m tcp --dport 8200 -j _NDM_SL_PROTECT

**Reply by Anonym-tsk** [reply]
Когда вдруг "процесс сам остановится", выполните команды и приложите вывод
`/opt/etc/init.d/S51nfqws status`

`ps | grep nfqws`

`iptables-save | grep 200`

Пока могу только предположить, что у вас флешка умирает на которой entware развернут.
Посмотрите логи роутера

**Comment by korolyuga** [comment]
status после перезапуска тут же stoped
-------
~ # ps | grep nfqws
 1220 nobody   13504 S    /opt/zapret/nfq/nfqws --user=nobody --dpi-desync-fwm
 4606 root      5400 S    grep nfqws
--------
~ # iptables-save | grep 200
-A POSTROUTING -o eth3 -p udp -m multiport --dports 443,50000:50099 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A POSTROUTING -o eth3 -p tcp -m multiport --dports 443 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A _NDM_IP_PROTECT -p tcp -m tcp --dport 8200 -j _NDM_SL_PROTECT

**Reply by Anonym-tsk** [reply, marked as answer]
`/opt/zapret/nfq/nfqws` - это не часть nfqws-keenetic
Удалите все лишнее

**Comment by korolyuga** [comment]
директорию удалил, а как теперь заменить этот процесс на нужный?

**Reply by Anonym-tsk** [reply]
Перезагрузите роутер, должно все заработать (если вы конечно никаких скриптов там не добавляли)

**Reply by korolyuga** [reply]
Вот так должно быть, верно?

~ # ps | grep nfqws
 7837 nobody     644 S    /opt/usr/bin/nfqws --daemon --pidfile=/opt/var/run/n
 7972 root      5400 S    grep nfqws
~ # iptables-save | grep 200
-A POSTROUTING -o eth3 -p udp -m multiport --dports 443,50000:50099 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A POSTROUTING -o eth3 -p tcp -m multiport --dports 443 -m connbytes --connbytes 1:8 --connbytes-mode packets --connbytes-dir original -m mark ! --mark 0x40000000/0x40000000 -j NFQUEUE --queue-num 200 --queue-bypass
-A _NDM_IP_PROTECT -p tcp -m tcp --dport 8200 -j _NDM_SL_PROTECT
~ # /opt/etc/init.d/S51nfqws status
Service NFQWS is running

**Reply by Anonym-tsk** [reply]
Да, похоже все работает

**Reply by korolyuga** [reply]
Спасибо огромное вам!!!!теперь не отваливается)))
и последний вопрос  - забыл перенаправить результаты блокчека в файл, может быть он автоматически формируется?
